docs/user/metacat/source/configuration.rst

```diff
@@ -28,99 +25,59 @@ You can always open the configuration screen from within Metacat by typing::
 
-.. Note::
+Starting from Metacat version 3.0.0 or later, **metacat.properties** no longer contains any custom
+settings that need to be backed up before a Metacat upgrade. Instead, custom settings are now saved
+to a file named ``metacat-site.properties`` that is located outside of the tomcat webapps directory,
+and so is not overwritten by deploying a new Metacat war file.
 
-   For Metacat version 3.0.0 or later, **metacat.properties** no longer contains any custom settings
-   that need to be backed up before a Metacat upgrade. Instead, custom settings are now saved to a
-   file named ``metacat-site.properties`` that is located outside of the tomcat webapps directory,
-   and so is not overwritten by deploying a new Metacat war file.
+ **Note:** If you are unable to access your Metacat admin ORCID iD and need to swap it out, instructions
+ for `Changing Authentication Configuration without Authentication`_ are included at the end of this
+ section.
 
 Authentication Configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+----------------------------
 Whether you are installing or upgrading the Metacat servlet, you will 
 automatically be sent to the Authentication Configuration page. You can also 
-reach the Authentication Configuration page from a running Metacat by typing::
+reach the Authentication Configuration page from a running Metacat by typing
+
+::
   
   http://<your_context_url>/admin
 
-Metacat uses either an internal password file or LDAP as its authentication mechanism.
-You can choose the authentication mechanism by selecting either AuthFile or AuthLdap class.
-We will only allow trusted partners to access the NCEAS LDAP server to ensure the security of our user base.
-If you are not in the trusted partner list, you may choose the internal password file authentication 
-or set up your own LDAP server. You also can define your own authentication mechanism by creating a Java 
-class that implements ``AuthInterface``.
+Starting from Metacat v3.0.0, only `ORCID authentication`_ is supported. In Metacat v2.19.0 and
+previous releases, an internal password file or LDAP was used as the authentication mechanism.
+Password-based and LDAP authentication has been deprecated. If you don't already have an account,
+registering for an ORCID is simple, please visit:
 
-Required configuration values for the password file authentication are:
+  http://orcid.org/
 
-  ::
-   
-    Authentication Class
-    Metacat Administrators
-    Users Management URL
-    Password File Path.
+.. _ORCID authentication: ./authinterface.html
 
-Required configuration values for LDAP authentication are:
-
-  ::
-   
-    Authentication Class
-    Metacat Administrators
-    Users Management URL
-    Authentication URL
-    Authentication Secure URL. 
+After signing up for an ORCID iD, you can then use it as an admin identity when configuring
+authentication.
 
+ **Reminder:** Your full ORCID iD includes "https://orcid.org/" not just the 16-digit ORCID iD:
```

Comment:
instead of:
~~**Reminder:** Your full ORCID iD includes "https://orcid.org/" not just the 16-digit ORCID iD:~~

I'd try to reduce confusion by saying:

**Important:** Even though your ORCID iD should typically be expressed as a full https URI (e.g. https://orcid.org/0000-0001-2345-6789), it is necessary to use **http-only orcid URIs** when entering them in the Metacat Administrator Interface: